### PR TITLE
Override github url 

### PR DESCRIPTION
**Pull request by wtrocki**
Reverts redhat-developer/app-services-architecture#10

### CODE REVIEW

**Comment by wtrocki** [comment]
Plugin provides metadata and it works locally but somehow fails on deployment:
https://jekyll.github.io/github-metadata/site.github/

Not sure why I done hack based on docs:
https://github.com/pages-themes/leap-day#overriding-github-generated-urls

So that will specify this value explicitly.

File: _config.yml [hunk 1]
```diff
@@ -55,6 +54,9 @@ collections:
 
 include:
  - CNAME
+ 
+ github:
```
Comment (tombentley) [inline _config.yml:58]: Probably worth a comment as to why we're doing this. E.g. a link to https://github.com/redhat-developer/app-services-architecture/pull/11 would be enough, I think.

**Review by tombentley — APPROVED** [review]

**Comment by wtrocki** [comment]
Still not sure if that is going to work. It always worked locally.

**Comment by tombentley** [comment]
Let's try it. I suppose that upgrading jekyll would be an alternative thing to try, but a bigger change than this.

**Comment by wtrocki** [comment]
Actually I have missed that info in logs when locally:

 >  GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.
 
 So proper fix is to not use that and hardcode values or use github api key

**Pull request merged**